py/travis-ci/u_boot_boardenv_qemu_x86_na.py

```diff
@@ -12,3 +12,11 @@
     "size": os.path.getsize(u_boot_bin),
     "crc32": hex(binascii.crc32(open(u_boot_bin, 'rb').read()) & 0xffffffff)[2:],
 }
+
+helloworld_bin = os.environ['UBOOT_TRAVIS_BUILD_DIR'] + "/lib/efi_loader/helloworld.efi"
```

Comment:
I guess I'll wait to apply this commit until I see the u-boot.git changes that implement this get checked in (or you let me know if I miss it).